ui/components/component-library/modal-content/__snapshots__/modal-content.test.tsx.snap

```diff
@@ -1,11 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`ModalContent should match snapshot 1`] = `
-<div>
-  <div
-    class="box mm-modal-content mm-modal-content--size-sm box--padding-4 box--flex-direction-row box--width-full box--background-color-background-default box--rounded-lg"
-  >
-    test
-  </div>
-</div>
-`;
+exports[`ModalContent should match snapshot 1`] = `<div />`;
```

Comment:
This seems funky. Check test

Reply:
Turns out this would be the expected result for the container snapshot. Have updated it to use getByTestId so it renders the wrapping div and className